Sources/WebUI/WebView.swift

```diff
@@ -28,11 +28,20 @@ public struct WebView {
     /// - Parameters:
     ///   - request: The initial request specifying the URL to load.
     ///   - configuration: The configuration for the new web view.
-    public init(request: URLRequest? = nil, configuration: WKWebViewConfiguration = .init()) {
+    public init(request: URLRequest? = nil, configuration: WKWebViewConfiguration) {
         self.initialRequest = request
         self.configuration = configuration
     }
 
+    /// Creates new WebView with default configuration.
+    /// - Parameters:
+    ///   - request: The initial request specifying the URL to load.
+    @MainActor
+    public init(request: URLRequest? = nil) {
+      self.initialRequest = request
+      self.configuration = WKWebViewConfiguration()
+    }
+
```

Comment:
In swift5, View is not given @MainActor, so giving @MainActor to an existing init will change its behavior. (It will generate a warning as shown in the image.)
I separated the two to minimize the impact as much as possible.

However, WKWebViewConfiguration.init is correct to call from @MainActor, so the warning should have been displayed. So I will add @MainActor and modify the implementation.

![スクリーンショット 2024-09-15 0 58 52](https://github.com/user-attachments/assets/cfa84e94-d9fd-4214-88e3-12ebedb59400)